### PR TITLE
pull request

### PR DESCRIPTION
**Pull request by mmarganski**
everything in /src/app.gateway.ts

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# compiled output
+/dist
+/node_modules
+
+# Logs
+logs
+*.log
+npm-debug.log*
```
Comment (WadasGrzegorz) [inline .gitignore]: it contains in ```*.log```

File: .prettierrc [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+  "singleQuote": true,
+  "trailingComma": "all"
```
Comment (WadasGrzegorz) [inline .prettierrc]: remove this file

Comment (WadasGrzegorz) [reply .prettierrc]: remove also package-lock.json if you are using yarn

**Review by WadasGrzegorz — CHANGES_REQUESTED** [review]

**Comment by jpudysz** [comment]
Please check your dependencies and devDependencies, they are invalid

File: package.json [hunk 3]
```diff
@@ -21,18 +21,17 @@
     "test:e2e": "jest --config ./test/jest-e2e.json"
   },
   "dependencies": {
+    "socket.io": "4.1.3"
```
Comment (jpudysz) [inline package.json]: .

File: src/app.gateway.ts [hunk 4]
```diff
@@ -1,137 +0,0 @@
...
-
-    @WebSocketServer() server: any
-    users: Map<string, Array<string>> = new Map() // socket.id => [userName, avatar]
-    rooms: Map<string, Array<string>> = new Map([[Rooms.All, []]]) // roomName => [... userIds]
-    privateRooms: Set<string> = new Set()
-    messages: Map<string, Array<string | Message>> = new Map([[Rooms.All, []]]) //  roomname => [...Message]
-    activeUsers: Set<string> = new Set()
-
-    @SubscribeMessage('getUsersList')
-    getUserList(client: Socket): void {
```
Comment (jpudysz) [inline src/app.gateway.ts:22]: as I said earlier, no need for void, you can remove it

File: src/app.gateway.ts [hunk 5]
```diff
@@ -1,137 +0,0 @@
...
-            client.join(room)
-            this.rooms.set(room, [...this.rooms.get(room), client.id])
-        }
-
-        return { event: 'messageHistory', data: this.messages.get(room) }
-    }
-
-    @SubscribeMessage('joinPrivateRoom')
-    joinPrivateRoom(client: Socket, userId: string): WsResponse {
-
```
Comment (jpudysz) [inline src/app.gateway.ts:73]: remove enter

File: src/app.gateway.ts [hunk 6]
```diff
@@ -1,137 +0,0 @@
...
-            this.rooms.set(roomName, [client.id, userId])
-            this.messages.set(roomName, [])
-        }
-
-        return { event: 'messageHistory', data: this.messages.get(roomName) }
-    }
-
-    @SubscribeMessage('sendMessage')
-    sendMessage(client: Socket, messageCall: MessageCall): WsResponse {
-
```
Comment (jpudysz) [inline src/app.gateway.ts:87]: aa

**Review by jpudysz — CHANGES_REQUESTED** [review]

**Review by jpudysz — APPROVED** [review]

**Review by WadasGrzegorz — APPROVED** [review]

**Pull request merged**